src/plugins/OktaPluginA11y.ts

```diff
@@ -78,9 +79,11 @@ export const init = (widget: OktaSignInAPI): void => {
           const isVisible = input.type !== 'password';
           button.type = 'button';
           if (isVisible) {
+            // @ts-ignore
```

Comment:
Need to fix TS error here 
`error TS2339: Property 'ariaLabel' does not exist on type 'HTMLButtonElement'.` 
that came up after adding `"src/plugins/*"` to  [tsconfig.json](https://github.com/okta/okta-signin-widget/pull/3546/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0)

Reply:
Looks like it's an issue with `typescript`'s DOM type defs.
Updating `typescript` from `~4.2` to `~4.4` resolves this issue.

Sadly can't easily align TS versions in root and `src/v3` (to `^4.9.5`) as there is an [issue](https://github.com/zerkalica/zerollup/issues/37) with `@zerollup/ts-transform-paths` plugin